packages/resizeImageHttp/index.js

```diff
@@ -15,14 +16,20 @@ const app = express();
 app.use(helmet());
 app.get('/', async (req, res) => {
   const { url } = req.query;
-  const { origin, referer } = req.headers;
-  if (!IS_TESTNET && origin && !origin.includes(ORIGIN_DOMAIN)) {
-    res.sendStatus(403);
-    return;
-  }
-  if (!IS_TESTNET && referer && !referer.includes(ORIGIN_DOMAIN)) {
-    res.sendStatus(403);
-    return;
+  const { referer } = req.headers;
+  if (referer) {
```

Comment:
Why drops the `origin` header checking?

Reply:
at least chrome does not send it when fetching img src, only referrer is sent
.... maybe useful for a canvas loaded img though, not sure

Reply:
But then keeping it should not harm, and some browser does not send `referer` header, so keeping the `origin` should in theory catch more cases?

Reply:
i doubt if they dont send `referer` would they send `origin`....
still we can keep it